pacman-rules/lib/rules.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Rules do
+  def eat_ghost?(power_pellet_active, touching_ghost) do
+     power_pellet_active && touching_ghost 
+  end
+
+  def score?(touching_power_pellet, touching_dot) do
+     touching_power_pellet || touching_dot 
+  end
+
+  def lose?(power_pellet_active, touching_ghost) do
+     not power_pellet_active && touching_ghost 
+  end
+
+  def win?(has_eaten_all_dots, power_pellet_active, touching_ghost) do
+     has_eaten_all_dots && not touching_ghost  || power_pellet_active 
```

Comment:
Firstly, there is an additional space which is not recommended. You could write this in a better way utilising the `lose?/2` function that you defined. Can you think of how to utilise that?